boards/arm/ebyte_e73_tbb_nrf52832/ebyte_e73_tbb_nrf52832-pinctrl.dtsi

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2022 morsisko
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&pinctrl {
+	uart0_default: uart0_default {
+		group1 {
+			psels = <NRF_PSEL(UART_TX, 0, 6)>,
+					<NRF_PSEL(UART_RX, 0, 8)>,
+					<NRF_PSEL(UART_RTS, 0, 5)>,
+					<NRF_PSEL(UART_CTS, 0, 7)>;
+		};
+	};
+
+	uart0_sleep: uart0_sleep {
+		group1 {
+			psels = <NRF_PSEL(UART_TX, 0, 6)>,
+			<NRF_PSEL(UART_RX, 0, 8)>,
+			<NRF_PSEL(UART_RTS, 0, 5)>,
+			<NRF_PSEL(UART_CTS, 0, 7)>;
+			low-power-enable;
```

Comment:
```suggestion
			psels = <NRF_PSEL(UART_TX, 0, 6)>,
				<NRF_PSEL(UART_RX, 0, 8)>,
				<NRF_PSEL(UART_RTS, 0, 5)>,
				<NRF_PSEL(UART_CTS, 0, 7)>;
		};
	};

	uart0_sleep: uart0_sleep {
		group1 {
			psels = <NRF_PSEL(UART_TX, 0, 6)>,
				<NRF_PSEL(UART_RX, 0, 8)>,
				<NRF_PSEL(UART_RTS, 0, 5)>,
				<NRF_PSEL(UART_CTS, 0, 7)>;
			low-power-enable;
```

Reply:
please align properly (make sure your editor has the right settings)

Reply:
Sorry missed it, hope it looks ok now